src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5656,6 +5656,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <!-- Get the resolved SDK reference items -->
   <Target Name="GetResolvedSDKReferences" DependsOnTargets="ResolveSDKReferences" Returns="@(ResolvedSDKReference)"/>
 
+  <!-- Target will return no results by default but will be overridden by other targets elsewhere.
```

Comment:
I don't think this comment is technically accurate.  This target isn't overridden anywhere, it's just that other targets may happen to populate `ReferencedNuGetPackages` as a side-effect, and we want to collect them as a return value if they do.